scripts/exgfs_atmos_grib2_special_npoess.sh

```diff
@@ -146,57 +133,44 @@ done
 ################################################################
 # Specify Forecast Hour Range F000 - F180 for GOESSIMPGRB files 
 ################################################################
-export SHOUR=000
+export SHOUR=0
 export FHOUR=180
-export FHINC=003
-if [[ "${FHOUR}" -gt "${FHMAX_GFS}" ]]; then
+export FHINC=3
+if (( FHOUR > FHMAX_GFS )); then
    export FHOUR="${FHMAX_GFS}"
 fi
 
 #################################
 # Process GFS PGRB2_SPECIAL_POST
 #################################
 
-for (( fhr=$((10#${SHOUR})); fhr <= $((10#${FHOUR})); fhr = fhr + FHINC )); do
+for (( fhr=SHOUR; fhr <= FHOUR; fhr = fhr + FHINC )); do
 
    fhr3=$(printf "%03d" "${fhr}")
 
    ###############################
    # Start Looping for the 
    # existence of the restart files
    ###############################
-   set +x
    export pgm="postcheck"
-   ic=1
-   while (( ic <= SLEEP_LOOP_MAX )); do
-      if [[ -f "${COM_ATMOS_GOES}/${RUN}.t${cyc}z.special.grb2if${fhr3}.idx" ]]; then
-         break
-      else
-         ic=$((ic + 1))
-         sleep "${SLEEP_INT}"
-      fi
-      ###############################
-      # If we reach this point assume
-      # fcst job never reached restart
-      # period and error exit
-      ###############################
-      if (( ic == SLEEP_LOOP_MAX )); then
-         echo "FATAL ERROR: Special goes grib file not available after max sleep time"
-         export err=9
-         err_chk || exit "${err}"
-      fi
-   done
-   set_trace
+   grib_file="${COM_ATMOS_MASTER}/${RUN}.t${cyc}z.goesmasterf${fhr3}.grb2"
+   if ! wait_for_file "${grib_file}" "${SLEEP_TIME}" "${SLEEP_LOOP_MAX}"; then
```

Comment:
`"${SLEEP_TIME}"` or `"${SLEEP_INT}"`?

Reply:
Should be `SLEEP_INT` like the one earlier in the script. Fixed.